test/Microsoft.AspNet.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -2,13 +2,18 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 #if DNX451
```

Comment:
It doesn't work on CoreClr. The ssl validation callback gets a X509Certificate and the constructor X509Certificate2(X509Certificate) to convert it doesn't exist (yet).

I created an issue: https://github.com/dotnet/corefx/issues/4510